Makefile

```diff
@@ -10,24 +10,34 @@ local:
 local-db:
 	docker-compose -f docker-compose.yml -f docker-compose.override.db.yml up -d db
 
+initialize-local-db:
+	docker-compose -f docker-compose.yml -f docker-compose.override.db.yml run backend bash -c "python runners/initialize_postgres.py"
+	echo "Performing load of initial Encompass data."
+	docker-compose -f docker-compose.yml -f docker-compose.override.db.yml run backend bash -c "python runners/load_representative_points.py -u -f 'data/sample/los-angeles-points.geojson'"
+	docker-compose -f docker-compose.yml -f docker-compose.override.db.yml run backend bash -c "python runners/load_addresses.py -f 'data/sample/mock-providers.csv'"
+
 rebuild:
 	docker-compose build --no-cache
 
-# Fetch data from S3 and load to Postgres.
-# State should be specified as a lowercase, two-letter abbreviation, e.g. 'ca'.
-# Example usage: make load_representative_points state='ca'
-S3_BUCKET='https://s3-us-west-1.amazonaws.com/network-adequacy/data-02-15-18/etl/output/'
-load_representative_points:
-	curl  --create-dirs -o 'data/representative_points.geojson' ${S3_BUCKET}$(state)'_representative_points.geojson'
-	docker-compose run backend bash -c "python runners/load_representative_points.py -f 'data/representative_points.geojson'"
-	rm data/representative_points.geojson
+load-local-state:
+	# Usage: make load-local-state state=ca filename=sample/los-angeles-points.geojson args="-c -u" (census_data and urban_data)
```

Comment:
Maybe just use the long names in `args=` so that you don't need to have them in parentheses? Would make it easier to copy this entire line.